src/main/java/seedu/tp/flashcard/FlashcardFactory.java

```diff
@@ -25,16 +48,20 @@ public Flashcard create(String flashcardType) throws UnrecognizedFlashcardTypeEx
         case "event":
```

Comment:
I'm thinking maybe we can keep `LOGGER.info("Received unknown flashcard type: " + flashcardType);` because this indicates an invalid input (maybe can also change the logging level to `warning`?) but remove the logging for other cases

Reply:
Yeah, I think that is a good idea. I will make the factory write to the same log as the flashcards as well. I don't think the code ever reaches the unknown flashcard in the factory right now because unknown commands are handled in the Parser, but logging this could be helpful if we change things in the future.